project/Dependencies.scala

```diff
@@ -5,8 +5,8 @@ import sbt._
 object Dependencies {
   val identityLibVersion = "3.240-C5"
   val awsVersion = "1.11.240"
-  val capiVersion = "17.22.1"
-  val faciaVersion = "3.3.8"
+  val capiVersion = "17.24.1"
```

Comment:
Probably the root cause of https://github.com/guardian/dotcom-rendering/issues/4343